mw-webapp/src/logic/wayPage/reportsTable/WayColumns.tsx

```diff
@@ -248,18 +261,10 @@ export const Columns = (props: ColumnsProps) => {
                     {isOwner &&
                     <TrashIcon
                       className={styles.icon}
-                      onClick={() => {
-
-                        /**
-                         * CallBack triggered on press ok
-                         */
-                        const onOk = () => deleteJobDone(jobDone.uuid);
-
-                        renderModalContent({
-                          description: jobDone.description,
-                          onOk,
-                        });
-                      }
+                      onClick={() => handleOnClick(
+                        `Are you sure that you want to delete jobDone "${jobDone.description}"?`,
```

Comment:
better to use object instead of multiple params

Reply:
Deleted wrapper and use object